studio/src/placeholders/mas-placeholders-item.js

```diff
@@ -0,0 +1,351 @@
+import { LitElement, html, nothing } from 'lit';
+import { STATUS_PUBLISHED, TAG_STATUS_DRAFT } from '../constants.js';
+import Store from '../store.js';
+import ReactiveController from '../reactivity/reactive-controller.js';
+import { MasRepository } from '../mas-repository.js';
+import { confirmation } from '../mas-confirm-dialog.js';
+import { showToast } from '../utils.js';
+import { FragmentStore } from '../reactivity/fragment-store.js';
+import { Placeholder } from '../aem/placeholder.js';
+
+class MasPlaceholdersItem extends LitElement {
+    static properties = {
+        placeholderStore: { type: Object, reflect: false },
+        editing: { type: Boolean, attribute: true },
+        disabled: { type: Boolean, attribute: true },
+        activeDropdown: { type: Boolean, attribute: 'active-dropdown' },
+        toggleEditing: { type: Function, reflect: false },
+        toggleDropdown: { type: Function, reflect: false },
+        updatePending: { type: Function, reflect: false },
+    };
+
+    createRenderRoot() {
+        return this;
+    }
+
+    constructor() {
+        super();
+        /** @type {FragmentStore} */
+        this.placeholderStore = null;
+        this.editing = false;
+        this.disabled = false;
+        this.activeDropdown = false;
+        this.toggleEditing = null;
+        this.toggleDropdown = null;
+        this.updatePending = null;
+
+        this.handleRteValueChange = this.handleRteValueChange.bind(this);
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+        this.reactiveController = new ReactiveController(this, [
+            this.placeholderStore,
+        ]);
+    }
+
+    updated(changedProps) {
+        super.updated(changedProps);
+
+        // Needed to avoid unsafeHtml in the rte-field
+        if (this.editing && this.placeholder.isRichText) {
+            const rteField = this.querySelector('rte-field');
+            if (!rteField.initialized) {
+                rteField.innerHTML =
```

Comment:
avoiding unsafeHtml is a valid concern. It actually makes me wonder if we also need to sanitize the content... 🤔